<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LyricsSearchDialog</class>
 <widget class="QDialog" name="LyricsSearchDialog">
  <property name="windowTitle">
   <string>Search for lyrics</string>
  </property>
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mainLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="artistLabel">
       <property name="text">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="artistEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
